binary-compat/test/src/test/scala/BinaryCompaTest.scala

```diff
@@ -23,6 +23,16 @@ class BinaryCompaTest {
     Config.setup("foo", Array(oldClasspath, newClasspath))
     val mima        = new MiMaLib(Config.baseClassPath)
     val allProblems = mima.collectProblems(oldClasspath, newClasspath)
-    Assert.assertEquals(allProblems, Nil)
+    val unexpectedDescriptions =
+      allProblems.iterator
+        .map(_.description("new"))
+        .map{x => println(x); x}
+        .filterNot(_ == "static method id(scala.collection.Iterable,scala.collection.generic.CanBuildFrom)scala.collection.Iterable in class org.example.Lib has a different signature in new version, where it is <A:Ljava/lang/Object;C::Lscala/collection/Iterable<Ljava/lang/Object;>;>(TC;Lscala/collection/generic/CanBuildFrom<Lscala/runtime/Nothing$;TA;TC;>;)TC; rather than <A:Ljava/lang/Object;C::Lscala/collection/Iterable<Ljava/lang/Object;>;>(TC;Lscala/collection/generic/CanBuildFrom<TC;TA;TC;>;)TC;")
+        .filterNot(_ == "method id(scala.collection.Iterable,scala.collection.generic.CanBuildFrom)scala.collection.Iterable in object org.example.Lib has a different signature in new version, where it is <A:Ljava/lang/Object;C::Lscala/collection/Iterable<Ljava/lang/Object;>;>(TC;Lscala/collection/generic/CanBuildFrom<Lscala/runtime/Nothing$;TA;TC;>;)TC; rather than <A:Ljava/lang/Object;C::Lscala/collection/Iterable<Ljava/lang/Object;>;>(TC;Lscala/collection/generic/CanBuildFrom<TC;TA;TC;>;)TC;")
```

Comment:
The, uhm, "standard" way to do this would be a `ProblemFilter`, which is `Problem => Boolean` and constructed with: https://github.com/lightbend/mima/blob/1788742d28e7ec073152fd900bd89ec618168b16/core/src/main/scala/com/typesafe/tools/mima/core/ProblemFilters.scala#L22